Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_botiboti/src/main.rs

```diff
@@ -0,0 +1,82 @@
+use num_bigint::{BigUint, ParseBigIntError};
+use std::io;
+
+const DAYS_IN_YEAR: u64 = 365;
+
+// convert input string to BigUint
+fn read_parse(inp: &str) -> Result<BigUint, ParseBigIntError> {
+    let age = inp.trim().parse::<BigUint>();
+    age
+}
+
+// turn age in years to days
+fn calc_days(age: &BigUint) -> BigUint {
```

Comment:
maybe this should be in a separate module and exported, pending slack discussion